duneanaobj/StandardRecord/SRMeta.h

```diff
@@ -26,6 +26,8 @@ namespace caf
       /// detector-dependent trigger type for the relevant readout window
       int triggertype = -1;
 
+      bool istriggered = true; ///< Was the event triggered by this detector?
+
```

Comment:
this seems redundant to me.  `triggertype = -1` seems like the same as `istriggered = false`, no?

Reply:
For me the trigger type was there to define the kind of trigger used (e.g. no trigger, random trigger, HE trigger, ...). For a given type of trigger, we'll want to run the trigger generation on selected events and see if this specific event was actually triggered or not. I think it would be confusing to add this information in the trigger type.

Reply:
I think there's something about this I don't understand.  How could you get reconstructed objects corresponding to something that didn't trigger?  Is Pandora currently being run on literally every energy deposit, and no trigger simulation being done?  Is it possible to get a situation like that in real data?  (If not, then I think we may need to find a way to connect this to the MC branch, rather than here in the reco)

Reply:
On real data, ofc this wouldn't apply. However, the FD reconstruction is, for now, run on every single event independently of any triggering scheme. Recently the trigger code has been added to LArSoft allowing to run the trigger simulation together with the reconstruction. The idea is at the simulation level to run both to be able to inform the user if the reco event they are trying to do physics with will even be triggered or not. So I think the information is very relevant.
It's a bit tricky to find the best location where to put that as this might be unwanted in the reco part for real data as you are pointing out. However this is not really "true information" either and comes from some trigger algorithm.

Reply:
Okay.  Let me think about this for a little bit, I'll come back here shortly (today).

Reply:
Thanks for explaining this.  It's a very different workflow from the ND cases I'm more familiar with, but that's a valuable lesson for me.  I agree this is retaining different info and we should preserve it.  The only request I'd make is that we make it `triggered` instead of `istriggered` to match the pattern for status bools elsewhere in the SR.  Perhaps also with a slightly more expanded doxygen comment:
```suggestion
      bool istriggered = true; ///< Was the event triggered by this detector?  (Useful in contexts where all simulated events are reconstructed, and triggering simulation applied separately)

```